src/ebay-pagination/pagination.tsx

```diff
@@ -83,19 +98,46 @@ const EbayPagination: FC<PaginationProps> = ({
 
     const createChildItems = (itemType: PaginationItemType): ReactElement[] => {
         let pageIndex = 0
+        const [firstDotFrom, firstDotTo] = pageRange.firstDotRange || []
+        const [secondDotFrom, secondDotTo] = pageRange.secondDotRange || []
+        const firstDotItems : ReactElement[] = []
+        const secondDotItems : ReactElement[] = []
 
         return Children.map(children, (item: ReactElement, index) => {
             const { type = 'page', current, disabled, href, children: text } = item.props
+            const isDot = page[index] === 'dots'
+            const key = `${id}-item-${index}`
+            const hide = page[index] === 'hidden'
             const newProps = {
                 type, current, disabled, href,
-                children: page[index] === 'dots' ? '…' : text,
+                children: isDot ? '…' : text,
                 pageIndex: type === 'page' ? pageIndex++ : undefined,
-                key: `${id}-item-${index}`,
-                hide: page[index] === 'hidden',
+                key,
+                hide,
                 onPrevious, onNext, onSelect, a11yPreviousText, a11yNextText,
                 ref: childPageRefs.current[index]
             }
+            // include hidden numbers & number of (...)itself
+            if ((hide || isDot) && itemType === 'page') {
+                const itemComponent = (
+                    <Item key={key} href={href} showTickSmall={false} onClick={updatePages}>{text}</Item>
+                )
+                if (index <= firstDotTo && index >= firstDotFrom) {
+                    firstDotItems.push(itemComponent)
+                } else if (index <= secondDotTo && index >= secondDotFrom) {
+                    secondDotItems.push(itemComponent)
+                }
+            }
 
+            if (itemType === 'page' && isDot) {
+                return (
+                    <li key={key}>
+                        <EbayFakeMenuButton className="pagination__item" text="…" borderless showDropdown={false}>
```

Comment:
To make it more accessible, you need to change the HTML structure for the overflow menu:

- Wrap the fake menu button with a span with class `pagination__item` and add attribute `role="separator"`. Here, you added the class `pagination__item` directly on the fake menu button itself (which is not correct), thus you cannot add `role="separator"` to the fake menu button.
- Remove the text '...' from the fake menu button. The fake menu button should have `aria-label="Menu"`

Checkout the HTML structure on eBay UI Core Marko: https://opensource.ebay.com/ebayui-core/?path=/story/navigation-disclosure-ebay-pagination--interactive&args=variant:overflow or Skin https://opensource.ebay.com/skin/#pagination on **Overflow Control for Pagination**

Reply:
will update

Reply:
First feedback is updated'...'

I cannot remove text '...', I think `EbayFakeMenuButton` doesn't support `variant='overflow'` yet

Reply:
You're right, EbayFakeMenuButton did not support overflow variant. 

But now it does, the PR https://github.com/eBay/ebayui-core-react/pull/169 has been merged today. Please sync you feature branch with the main.

Reply:
updated